llvm/include/llvm/IR/PassManager.h

```diff
@@ -245,24 +245,27 @@ class PassManager : public PassInfoMixin<
     return PA;
   }
 
-  // FIXME: Revert to enable_if style when gcc >= 11.1
```

Comment:
https://www.llvm.org/docs/GettingStarted.html#getting-a-modern-host-c-toolchain says we still support some gcc <11 (this should be a separate PR anyway)

Reply:
This was introduced in https://github.com/llvm/llvm-project/commit/b361b5369ed4bb186f2a11459b1bc1d54d73a33e to distinguish machine function passes and ir passes. `MachinePassModel` is removed now, so we can revert to `enable_if` form.

Reply:
ah sorry, I was mixing up enable_if_t and enable_if